### ISSUE TITLE
Force ipv4 on a dual stack machine.

### ISSUE THREAD

**Issue by andersbs** [google/recaptcha#44]
I need to force ipv4 requests for recaptcha on a dual stack machine. Right now I do this using a copy of the Post RequestMethod by doing something like:

``` php
$host = gethostbyname('www.google.com';
$uri = sprintf('https://%s/recaptcha/api/siteverify', $host);
return file_get_contents($uri, false, $context);
```

Is this something worth submitting a PR for or would it be better as an option for the CURL RequestMethod I saw someone working on?

**Comment by rowan-m** [comment]
The answer to "Is this something worth submitting a PR" is almost always, "Yes, please!" I've merged #39 now, so feel free to look at implementing something that allows users to set options on the request method and then I can jump in for review and discussion. Thank you!

**Comment by rowan-m** [comment]
Closing super old issues. Please re-raise if still relevant.
